.github/workflows/gui-checks.yml

```diff
@@ -75,11 +77,12 @@ jobs:
 
       - name: 📝 Annotate Code Linting Results
         if: always()
+        id: eslint-annotate
         continue-on-error: true
-        uses: ataylorme/eslint-annotate-action@v3
+        uses: MrFlashAccount/eslint-annotate-action@v3
```

Comment:
We shouldn't be adding dependencies to personal GitHub projects. If we need to fork then we should fork it into the Enso GitHub space.

But also why are we forking this project? What is the plan for keeping up to date with the source?

Reply:
It hasn’t been updated for years. I forked it to fix a few issues. If you’d like, we can transfer it to Enso, but I’m not sure I have the necessary permissions.

Reply:
Happy to create an Enso Repo for this.

Reply:
@jdunkerley the repo is transferred, will update the PR later